WORKSPACE

```diff
@@ -126,7 +79,7 @@ load("@build_bazel_rules_typescript//:defs.bzl", "ts_setup_workspace", "check_ru
 ts_setup_workspace()
 
 # 0.16.0: tsc_wrapped uses user's typescript version & check_rules_typescript_version
-check_rules_typescript_version("0.16.0")
+check_rules_typescript_version("0.15.0")
```

Comment:
Needs to be changed after bazelbuild/rules_typescript#256 is merged and released.

Reply:
done